Gemfile

```diff
@@ -1,5 +1,5 @@
 source 'https://rubygems.org'
-ruby '2.4.1'
+ruby '2.5.1'
```

Comment:
please exclude `Gemfile` from PR

Reply:
ok
Shall i raise the PR again ?

Reply:
You can either raise a new PR or remove the file from this PR.